Cargo.toml

```diff
@@ -25,7 +25,7 @@ all = "warn"
 
 [workspace.lints.clippy]
 all = { level = "warn", priority = -1 }
-missing-const-for-fn = "warn"
+missing-const-for-fn = "allow" # TODO: https://github.com/rust-lang/rust-clippy/issues/14020
```

Comment:
this has been driving me insane for the last two days

Reply:
unfortunate that it's very unstable